client/proposer/service_test.go

```diff
@@ -170,26 +181,11 @@ func TestCollationWitInDiffPeriod(t *testing.T) {
 	fakeProposer.p2p.Broadcast(&tx)
```

Comment:
Can you change this whole chunk to something like this 
```
	for i := 0; i < 5; i++ {
		node.CommitWithBlock()
	}
	fakeProposer.Start()

	for i := 0; i < 5; i++ {
		node.CommitWithBlock()
	}

	fakeProposer.p2p.Broadcast(&tx)
```

Reply:
Sure, done.